src/style.css

```diff
@@ -0,0 +1,138 @@
+body,
+p {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--4 > .stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 > .stars__star:nth-of-type(n + 2) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+}
```

Comment:
<img width="469" alt="Screenshot 2022-07-26 at 09 24 47" src="https://user-images.githubusercontent.com/94289038/180937971-ea6eff40-590b-4eee-af9c-a8af7182dc05.png">
what will happen if I change modifier through js - it should still work for all modifiers, as in `stars` task